abcix/adapter/adapter.go

```diff
@@ -154,7 +154,7 @@ func (app *adaptedApp) DeliverBlock(req abcix.RequestDeliverBlock) (resp abcix.R
 	adaptGeneralization(&req, &resp, "abci.RequestEndBlock", app.abciApp.EndBlock)
 	endEvents := resp.Events
 
-	resp.Events = make([]abcix.Event, len(beginEvents)+len(endEvents))
+	resp.Events = make([]abcix.Event, len(beginEvents))
```

Comment:
nice we found a bug! instead of using `copy` then `append` how about `append` all together?

```golang
allEvents := make([]abcix.Event, 0, len(beginEvents)+len(endEvents)) 
allEvents = append(allEvents, beginEvents...)
allEvents = append(allEvents, endEvents...)
resp.Events = allEvents
```